test/unit/linear_algebra/test_error_function.cpp

```diff
@@ -0,0 +1,64 @@
+
+#include <cfloat>
+#include <cstddef>
+#include <cstdlib>
+#include <gtest/gtest.h>
+#include <tuvx/linear_algebra/linear_algebra.hpp>
+#include <vector>
+
+const std::size_t size = 10; // size of the system to test
+const int norm_order = 2;    // L2 norm for computing error
```

Comment:
If we switch to use the relative error, probably we do not need the `norm_order` variable in your test any more. If so, please clean up all the code and comments about it.

Reply:
@sjsprecious I have removed everything in reference to `norm_order`.